### PR TITLE
Add grafana integration docs

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/istio/istio/issues/22250

### CODE REVIEW

File: content/en/docs/ops/integrations/grafana/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Grafana
+description: Information on how to integrate with Grafana to set up Istio dashboards.
+weight: 20
+keywords: [integration,grafana]
+---
+
+[Grafana](https://grafana.com/) is an open source monitoring solution that can be used to configure dashboards for Istio. This allows monitoring the health of Istio itself, as well as of applications within the mesh.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:8]:
```suggestion
[Grafana](https://grafana.com/) is an open source monitoring solution that can be used to configure dashboards for Istio. You can use Grafana to monitor the health of Istio and of applications within the service mesh.
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Configuration
+
+While you can build your own dashboards, Istio offers a set of preconfigured dashboards for all of the most important metrics in the mesh.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:12]:
```suggestion
While you can build your own dashboards, Istio offers a set of preconfigured dashboards for all of the most important metrics for the mesh and for the control plane.
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+* [Mesh Dashboard](https://grafana.com/grafana/dashboards/7639) provides an overview of all services in the mesh.
+* [Service Dashboard](https://grafana.com/grafana/dashboards/7636) provides a detailed breakdown of metrics for a service.
+* [Workload Dashboard](https://grafana.com/grafana/dashboards/7630) provides a detailed breakdown of metrics for a workload.
+* [Performance Dashboard](https://grafana.com/grafana/dashboards/11829) monitors the resource usage of the mesh.
+* [Control Plane Dashboard](https://grafana.com/grafana/dashboards/7645) monitors the health and performance of the control plane.
+
+There are a few ways to configure Grafana to use these dashboards:
+
+### Use the built in Grafana deployment
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:22]:
```suggestion
### Use the built-in Grafana deployment
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+To deploy the built in Grafana, follow the [Install Guide](/docs/setup/install/istioctl/) and pass `--set values.grafana.enabled` during installation.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:24]:
```suggestion
The built-in deployment of Grafana is bundled with all of the Istio dashboards already installed. To deploy the built-in Grafana instance, follow the steps in [Customizable Install with Istioctl](/docs/setup/install/istioctl/) to install Istio and pass `--set values.grafana.enabled=true` during installation.
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+This is intended for new users quickly getting started, but does not offer advanced customization like persistence or authentication. However, it comes bundled with all of the Istio dashboards already installed.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:26]:
```suggestion
This built-in deployment of Grafana is intended for new users quickly getting started. However, it does not offer advanced customization, like persistence or authentication.
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+### Import from `grafana.com`
+
+Using the links above, you can quickly import these dashboards to your Grafana instance. This can be done through the [`Import` button in the UI](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard). Please note you will need to select a Prometheus data source to use when importing.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:30]:
```suggestion
To quickly import the Istio dashboards to an existing Grafana instance, you can use the [**Import** button in the Grafana UI](https://grafana.com/docs/grafana/latest/reference/export_import/#importing-a-dashboard) to add the dashboard links above. When you import the dashboards, note that you must select a Prometheus data source.
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+This can also be done through a script to import all dashboards at once. For example:
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:32]:
```suggestion
You can also use a script to import all dashboards at once. For example:
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+{{< text plain >}}
+# Address of grafana
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:35]:
```suggestion
# Address of Grafana
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+GRAFANA_HOST="http://localhost:3000"
+# Login credentials, if authentication is used
+GRAFANA_CRED="USER:PASSWORD"
+# The name of the prometheus data source to use
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:39]:
```suggestion
# The name of the Prometheus data source to use
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+GRAFANA_DATASOURCE="Prometheus"
+# The version of Istio we are deploying
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:41]:
```suggestion
# The version of Istio to deploy
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+        -H "Content-Type: application/json" \
+        -d "{\"dashboard\":$(cat /tmp/dashboard.json),\"overwrite\":true, \
+            \"inputs\":[{\"name\":\"DS_PROMETHEUS\",\"type\":\"datasource\", \
+            \"pluginId\":\"prometheus\",\"value\":\"$GRAFANA_DATASOURCE\"}]}" \
+        $GRAFANA_HOST/api/dashboards/import
+    echo -e "\nDone\n"
+done
+{{< /text >}}
+
+Please note that there will be a new revision of the dashboard for each version of Istio. It is recommended that you select the appropriate revision to ensure compatibility.
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:59]:
```suggestion
{{< tip >}}
A new revision of the dashboards is created for each version of Istio. To ensure compatibility, it is recommended that you select the appropriate revision for the Istio version you are deploying.
{{< /tip >}}
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+### Implementation specific methods
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:61]:
```suggestion
### Implementation-specific methods
```

File: content/en/docs/ops/integrations/grafana/index.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+
+There are many ways to install and configure Grafana which all have different ways to configure dashboards. To use these, please refer to the documentation of the installation method you have chosen. For example:
```
Comment (Rachael-Graham) [inline content/en/docs/ops/integrations/grafana/index.md:63]:
```suggestion
Grafana can be installed and configured through other methods. To import Istio dashboards, refer to the documentation for the installation method. For example:
```

**Comment by howardjohn** [comment]
@Rachael-Graham can you sign the CLA?

**Comment by Rachael-Graham** [comment]
@googlebot I fixed it.

**Pull request merged**